scripts/shared/clusters.sh

```diff
@@ -7,14 +7,16 @@ DEFINE_string 'k8s_version' '' 'Version of K8s to use'
 DEFINE_string 'globalnet' 'false' "Deploy with operlapping CIDRs (set to 'true' to enable)"
 DEFINE_string 'registry_inmemory' 'true' "Run local registry in memory to speed up the image loading."
 DEFINE_string 'cluster_settings' '' "Settings file to customize cluster deployments"
+DEFINE_string 'lighthouse' 'false' "Deploy with service discovery enabled (set to 'true' to enable)"
```

Comment:
Clusters just creates the multi cluster environment, is it even pertinent at this stage? I don't see any changes to the script itself other than adding this flag..

Reply:
Yes, wasn't sure if this will be needed so added it. Didn't notice it wasn't being used anywhere. Will remove this.